leapsectz/leapsectz.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package leapsectz is a utility package for obtaining leap second
+// information from the system timezone database
+
+package leapsectz
+
+import (
+	"encoding/binary"
+	"errors"
+	"os"
+	"time"
+)
+
+const file = "/usr/share/zoneinfo/right/UTC"
+
+var badData = errors.New("malformed time zone information")
```

Comment:
nit: From the Go style errors are called with Err in front, e.q errBadData https://blog.golang.org/go1.13-errors